src/git-command-manager.ts

```diff
@@ -188,22 +181,10 @@ class GitCommandManager {
   async lfsFetch(ref: string): Promise<void> {
     const args = ['lfs', 'fetch', 'origin', ref]
 
```

Comment:
Refactored most of the lines below into a retry helper. I needed to add retry logic to a new place.